### PR TITLE
Fix for cached reference genome selector

### PR DESCRIPTION
**Pull request by bollig**
http://dev.list.galaxyproject.org/no-reference-genome-list-in-variant-selector-tool-td4666145.html

A little old, but worth merging.

### CODE REVIEW

File: tool_collections/gatk/variant_filtration/variant_filtration.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <param name="input_variant" type="data" format="vcf" label="Variant file to annotate" help="-V,--variant &amp;lt;variant&amp;gt;" />
         <param name="ref_file" type="select" label="Using reference genome" help="-R,--reference_sequence &amp;lt;reference_sequence&amp;gt;">
           <options from_data_table="gatk_picard_indexes">
-            <filter type="data_meta" key="dbkey" ref="input_variant" column="dbkey"/>
+            <!-- <filter type="data_meta" key="dbkey" ref="input_variant" column="dbkey"/> does not yet work in a repeat...-->
```
Comment (nsoranzo) [inline tool_collections/gatk/variant_filtration/variant_filtration.xml]: It's not inside a `<repeat>` here.

File: tool_collections/gatk/variant_select/variant_select.xml [hunk 2]
```diff
@@ -106,7 +106,7 @@
         <param name="input_variant" type="data" format="vcf" label="Variant file to select" help="-V,--variant &amp;lt;variant&amp;gt;" />
         <param name="ref_file" type="select" label="Using reference genome" help="-R,--reference_sequence &amp;lt;reference_sequence&amp;gt;">
           <options from_data_table="gatk_picard_indexes">
-            <filter type="data_meta" key="dbkey" ref="input_variant" column="dbkey"/>
+            <!-- <filter type="data_meta" key="dbkey" ref="input_variant" column="dbkey"/> does not yet work in a repeat...-->
```
Comment (nsoranzo) [inline tool_collections/gatk/variant_select/variant_select.xml]: Same here.

**Comment by bgruening** [comment]
@blankenberg @davebx can you comment on this one, please.

**Comment by bgruening** [comment]
@bollig can you remove please the comment. It's wrong afaik.

**Comment by bgruening** [comment]
ping @bollig

**Comment by bgruening** [comment]
Thanks @bollig!

**Pull request merged**